src/reducer.js

```diff
@@ -55,6 +55,12 @@ function reducer(
         userL0s: _.uniqBy(_.map(userL1s, "parent"), "uuid"),
         userL1s,
       };
+    case "LOCATION_USER_DISTRICTS_CLEAR":
+      return {
+        ...state,
+        userL0s: [],
```

Comment:
I see that you clean this state attribute using log, during our prior discussion you've mentioned that userL0s and userL1s are not part of initial state, should we add them?

Reply:
@dborowiecki I tried to clear state attribute by adding them to initial state, but it wasn't working properly. So I used dispatch, it works like intended, so I didn't want to spend much time on doing research why the previous solution cause some issues. But if we have to know the reason, just let me know, I will take care of it.

Reply:
Thanks, as it is not mandatory from a Redux perspective to initialize an attribute in initial state in the reducer - we can keep it that way.